documentation/src/main/wiki/BestPractices.md

```diff
@@ -0,0 +1,218 @@
+## Best practices and tips for writing AET test scripts
+
+#### Learn the basics of Selenium WebDriver and understand how to use this knowledge when working with AET
+AET interacts with web pages through WebDriver so it's good to know how to use WebDriver. 
+Knowing about it's limitations and what it requires from test scripts to work properly can help a lot when 
+writing test scripts for AET. It's especially important to understand that in order to get stable tests 
+AET scripts should include smartly located wait operations, just like WebDriver test scripts do.
+
+#### Read the documentation
+AET currently has a lot of useful commands described in the GitHub docs. 
```

Comment:
Maybe let's link here to [Features](https://github.com/Cognifide/aet/wiki/Features) with `[[Features|Features]]`.

Reply:
Will do.